core/backend/src/test/integration/IModelWrite.test.ts

```diff
@@ -963,131 +966,87 @@ describe("IModelWriteTest (#integration)", () => {
     iModel.close();
   });
 
-  it.skip("should be able to upgrade a briefcase with an older schema", async () => {
+  it("should be able to upgrade a briefcase with an older schema", async () => {
     const projectName = "iModelJsIntegrationTest";
     const projectId = await HubUtility.queryProjectIdByName(managerRequestContext, projectName);
 
     /**
      * Test validates that -
      * - User "manager" upgrades the BisCore schema in the briefcase from version 1.0.0 to 1.0.10+
-     * - User "super" fails to upgrade while the above upgrade in process
-     * - User "manager" can push the changes made to the schemas up to iModelHub
-     * - User "super" can get the upgrade "manager" made as change sets without requiring schema locks
+     * - User "super" can get the upgrade "manager" made
      */
 
     /* Setup test - Push an iModel with an old BisCore schema up to the Hub */
     const pathname = IModelTestUtils.resolveAssetFile("CompatibilityTestSeed.bim");
     const hubName = HubUtility.generateUniqueName("CompatibilityTest");
-    const iModelId = await HubUtility.pushIModel(managerRequestContext, projectId, pathname, hubName);
+    const iModelId = await HubUtility.pushIModel(managerRequestContext, projectId, pathname, hubName, true);
 
-    /* User "manager" upgrades the briefcase */
+    // Download two copies of the briefcase - manager and super
     const args: RequestNewBriefcaseProps = {
       contextId: projectId,
       iModelId,
     };
-    await BriefcaseManager.downloadBriefcase(managerRequestContext, args);
+    const managerBriefcaseProps: LocalBriefcaseProps = await BriefcaseManager.downloadBriefcase(managerRequestContext, args);

```

Comment:
I dislike these unnecessary type annotations. It looks like you're trying to override the return type of the method.

Reply:
Removed this across all occurrences in the backend.